packages/webpack-configs/src/transformers/moduleRules.ts

```diff
@@ -3,15 +3,21 @@ import type { Configuration, RuleSetRule, RuleSetUseItem } from "webpack";
 
 export type ModuleRuleMatcher = (moduleRule: RuleSetRule | RuleSetUseItem, index: number, array: RuleSetRule[] | RuleSetUseItem[]) => boolean;
 
+export type WithModuleRuleMatcherInfo = {
```

Comment:
export type WithModuleRuleMatcherInfo

Not sure i would export this interface since it seems pretty internal